configuring-sys-image-registry.html.md.erb

```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml
+        sed -i 's/image: registry.pivotal.io/image: 10.195.35.68/g' relocated_images.yml
```

Comment:
Presumably the IP address here needs to be replaced with some variant of PRIVATE_REPOSITORY?